packages/grid/data-grid/src/tests/pagination.DataGrid.test.tsx

```diff
@@ -102,7 +102,7 @@ describe('<DataGrid /> - Pagination', () => {
       expect(onPageChange.callCount).to.equal(2);
```

Comment:
can you add the assertion on the call argument please

Reply:
:facepalm: I copied another test without updating the assertions